tests/sentry/uptime/endpoints/test_organization_uptime_stats.py

```diff
@@ -59,6 +61,40 @@ def test_simple(self):
             ]
         }
 
+    @freeze_time(datetime(2025, 1, 21, 19, 4, 18, tzinfo=timezone.utc))
+    @override_options({"uptime.date_cutoff_epoch_seconds": 1736881457})
```

Comment:
Good job on freezing like this, hopefully this won't end up unstable.

Will we hit retention limits when this ends up more than 90 days out?

Reply:
i've gone ahead and used a dynamic freezetime on the class, so this should be good.